source/scenario/newmission3task3.vwf.yaml

```diff
@@ -145,13 +145,26 @@ children:
               - enabled
               - true
 
-        highlightEndTile:
+        highlightEndTile3_3:
           triggerCondition:
           - onScenarioStart:
           actions:
           - callOutObjective:
             - [ -17, 26 ]
 
+        failOnMissingBlock_3e:
+          group: successOrFailure
+          priority: 3
+          triggerCondition:
+          - or:
+            - onBlockMissing:
+              - rover
+              - controls_if_nomut
+          actions:
+          - scenarioFailure:
+            - incomplete
+            - "You must use an if block."
```

Comment:
"You need to put your conditional loops inside of an if block, so that they will only execute if you need to move south."